Source/Core/VideoCommon/PerformanceTracker.h

```diff
@@ -89,16 +78,15 @@ class PerformanceTracker
   // Queue + Running Total used to calculate average dt
   DT m_dt_total = DT::zero();
   std::array<DT, MAX_DT_QUEUE_SIZE> m_dt_queue;
-  std::size_t m_dt_queue_begin = 0;
-  std::size_t m_dt_queue_end = 0;
+  u64 m_dt_queue_begin = 0, m_dt_queue_end = 0;
```

Comment:
Please don't put multiple variables on a single line. Also, why the change from `size_t` to `u64`?

Reply:
because of the way that I do circular buffer it is incredibly important that the integers are unsigned. so making them u64 just makes that clear.

Reply:
I'll add your changes tomorrow but i must sleep now. thanks for the review.